infra/modules/service/outputs.tf

```
@@ -0,0 +1,4 @@
+output "public_endpoint" {
+  description = "The DNS name of the load balancer that is created in the service module."
```

we're in the service module already, so the description doesn't need to mention the service module
```suggestion
  description = "The public endpoint of the load balancer"
```